Anishchenko_V/HW10/Git-outside/04 Classy classes.py

```diff
@@ -0,0 +1,8 @@
+class Person:
```

Comment:
look at this more optimized version of code

class Person:
    def __init__(self, name, age):
        self.name = name
        self.age = age

    @property
    def info(self):
        self.info = f"{self.name}'s age is {self.age}"
        return self.info